### ISSUE TITLE
Support for load_metric has been removed in datasets@3.0.0

### ISSUE THREAD

**Issue by usmanyaqoob49** [nlp-with-transformers/notebooks#153]
So basically in chapter 8 we have used "from datasets import load_metric" but now in the latest version of the datasets they have removed the load_metric and now you have to use the evaluate module for that.
So replace the mentioned line with:

import evaluate
accuracy_score= evaluate.load('accuracy')

**Comment by Emmanuel-Ibekwe** [comment]
Hello bro. Unrelated to your issue. Pls, I've got an issue with some section of chapter 4. Here's the link the kaggle notebook: https://www.kaggle.com/code/immanuelibekwe/fork-of-nlp-chapter-4
The trainer.train() runs without the gpu or cpu showing any indication of usage.

Also, when I run the code below, the cpu ui indicates usage instead of the gpu. What could be the cause:
import torch

device = torch.device("cuda" if torch.cuda.is_available() else 'cpu')
print(device.type)
xlmr_model = (XLMRobertaForTokenClassification
              .from_pretrained(xlmr_model_name, config=xlmr_config).to(device))

print(xlmr_model.device)

I also ran into some problems using the Trainer api in chapter 2 but no one has responded to my issue for 3 weeks now

**Comment by usmanyaqoob49** [comment]
@Emmanuel-Ibekwe Are you sure you have GPU available?
If yes make sure following things:

- The environment isn't set up correctly. Maybe CUDA isn't properly installed, or PyTorch isn't the GPU version. You should check if torch.cuda.is_available() returns True. If not, they need to install the CUDA-compatible PyTorch.
- Make sure your data and model are on same device.

Even if nothing happens try Google Collab GPU.
